test/grub2_efi_test.rb

```diff
@@ -4,7 +4,9 @@
 
 describe Bootloader::Grub2EFI do
   before do
-    allow(::CFA::Grub2::Default).to receive(:new).and_return(double("GrubDefault").as_null_object)
+    default = double("GrubDefault").as_null_object
+    allow(default).to receive(:timeout) # collision between timeout method of double and attr
```

Comment:
what is interesting that `default.timeout` work as expected but `default.send(:timeout)` raise exception, so I expect problem with some private method.